backend/rent.py

```
@@ -96,4 +96,4 @@ def signup():
     return render_template('signup.html')
 
 if __name__ == '__main__':
-    app.run(debug=True)
\ No newline at end of file
+    app.run(host='0.0.0.0',port=5000,debug=True)
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)